### ISSUE TITLE
[feature request] span single background over entire window

### ISSUE THREAD

**Issue by RicArch97** [jovanlanik/gtklock#28]
I'm using a multi monitor setup, and i like to make my screen locker extra fancy.

I came up with the idea to create a screenshot of both outputs combined and apply blur to it using imagemagick, and then use this as wrapper for launching gtklock where the image is provided to it upon launch. 

I did however not find a way to span this background over gtklock, so a singular image covers both outputs. I'm guessing this is the best way to do it, as setting specific backgrounds per output is probably not easy to implement.

So basically i have a 2560x1440 and 3440x1440 next to each other, so my screenshot is 6000x1440. One big canvas.

**Comment by RicArch97** [comment]
dumb me... i completely missed the per-output styling section in the wiki... apologies!

**Comment by jovanlanik** [comment]
Glad it worked out.
